pkg/hubble/observer/observeroption/defaults.go

```diff
@@ -14,10 +14,12 @@
 
 package observeroption
 
+import "github.com/cilium/cilium/pkg/hubble/container"
+
 // Default serves only as reference point for default values. Very useful for
 // the CLI to pick these up instead of defining own defaults that need to be
 // kept in sync.
 var Default = Options{
-	MaxFlows:      131071, // 2^17-1
+	MaxFlows:      container.Capacity4095, // 4095
```

Comment:
From what I understand from commit log, this value is overwritten by the CLI defaults? Isn't that in contradiction with the comment above, or am I missing something?

In any case it would be great to have the value only defined at a single location.

Reply:
That's a very good point, I will fix that.